components/expenses/Expense.js

```diff
@@ -122,6 +135,16 @@ class Expense extends React.Component {
     this.setState(newState);
   }
 
+  handleUnapproveExpense = async id => {
+    try {
+      await this.props.unapproveExpense(id);
+      this.setState({ showUnapproveModal: false });
+    } catch (err) {
+      console.error(err);
+      this.setState({ showUnapproveModal: false });
+    }
```

Comment:
Not a good practice to hide the error without informing the user

Reply:
The error display in the dashboard is current not nice, I've opened  [#2567](https://github.com/opencollective/opencollective/issues/2567) to deal with this. There are other methods within the host dashboard that hides error, I'll handle everything together.

Reply:
Great